code/modules/organs/organ_internal.dm

```diff
@@ -15,6 +15,40 @@
 	robotic_name = "circulatory pump"
 	robotic_sprite = "heart-prosthetic"
 
+/obj/item/organ/stomache
+	name = "stomache"
+	icon_state = "stomache"
+	gender = PLURAL
+	organ_tag = "stomache"
+	parent_organ = "groin"
+	robotic_name = "gas exchange system"
+	robotic_sprite = "heart-prosthetic"
+	var/vomit_timer = FALSE
+
+/obj/item/organ/stomache/Initialize()
+	START_PROCESSING(SSfast_process, src)
```

Comment:
Is there not a better way to do this? Such as checking if it's too full only when its fullness is modified?